CONTRIBUTING.md

```diff
@@ -51,21 +51,21 @@ Make sure you have the following dependencies installed first:
 You need to have commit rights to the GitHub repository to publish a release.
 
 1. Update the version number in the `package.json` file.
-2. Update the `CHANGELOG.md` with the changes containted in the release.
+2. Update the `CHANGELOG.md` with the changes contained in the release.
 3. Commit the changes to master and push to GitHub.
-4. Create a release branch locally that follows the convention v(major).(minor).x, leaving the patch as `x`. For example:
+4. Create a tag locally that follows the convention v(major).(minor).x, leaving the patch as `x`. For example:
 
    ```bash
-   git checkout -b v3.3.x
+   git tag v3.3.0
```

Comment:
this line is a bit confusing, saying 'leaving the patch as `x`' but then it....doesn't.
Would it make more sense for it to be `git tag v3.3.x`?